pkgs/build-support/writers/default.nix

```diff
@@ -76,6 +76,8 @@ rec {
       contentPath = content;
     }) ''
       ${compileScript}
+      ${lib.optionalString strip
+         "${pkgs.binutils-unwrapped}/bin/strip --strip-unneeded $out"}
```

Comment:
lib is globally in scope (with lib; on line 3) so you don't need to prefix lib here

Reply:
I prefer explicit qualification, if you insist I will change it though.
(also one day we might actually deprecate `with` in its current form, because it prevents a proper type system for the nix language, but I have to agree that's too far off to really matter atm)